process/phases.md

```diff
@@ -0,0 +1,150 @@
+# WebAssembly W3C Process
+
+This file describes how feature proposals will progress through
+the standardization process.
```

Comment:
I think we should require a proposal repo for everything that adds or changes a "feature". Criterions for that could be:

* Adds an opcode or other pieces of abstract syntax
* Changes or extends the binary format
* Changes or extends the text format
* Requires adding or changing tests
* Requires extending or changing the reference interpreter.

Any of these implies that all conforming engines are forced to be changed, which justifies going through a bit of process.

A "fix" on the other hand is something that e.g. is merely a clarification, and does not necessarily require implementations to change. That's perfectly doable as a simple PR.